apps/storybook-react/stories/Search.stories.tsx

```diff
@@ -33,19 +38,19 @@ const StyledSearch = styled(Search)`
 
 export default {
   title: 'Components/Search',
-  component: Search,
```

Comment:
Shouldn't remove the component key?

Reply:
We have no `argsTable` as all props are "native" input props. 
So I was left with: 
![image](https://user-images.githubusercontent.com/1070981/97585544-8abb1680-19f9-11eb-9422-47f23dfc2169.png)

We could start adding value, defaultValue and other native props to table, but we would have to just replicate whats already defined in React, which didn't seem right. 

To remove the "empty" argsTable, you are supposed to remove the `component` key, according to storybook docs.

Reply:
I see!

Reply:
I do see that [Material still documents "native" props](https://material-ui.com/api/text-field/). Which maybe we should also do for better clarity on how its supposed to be used. But maybe thats a discussion for when we start documentation work #449

Reply:
Yes, I was thinking of the same with my selection controls, feels like some important stuff was missing in the props table because it was native